notebooks/datasets/nldas_time_series.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # The bounding box should be passed to the geojson param as a geojson Feature or FeatureCollection
</pre><p>Provide a high level chapter title for this section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>